packages/api/app/Models/TechnologyReview.js

```diff
@@ -0,0 +1,19 @@
+/** @type {typeof import('@adonisjs/lucid/src/Lucid/Model')} */
+const Model = use('Model');
+
+class TechnologyReview extends Model {
+	static boot() {
+		super.boot();
+		this.addTrait('Params');
+	}
+
+	technology() {
+		return this.belongsTo('App/Models/Technology');
```

Comment:
Até hoje eu me confundo nesse negócio de `belongsTo`, `belongsToMany`, `hasMany`...e sempre vou olhar na documentação :smiley: .

Reply:
O belongsTo funciona com o hasMany e hasOne, indica que a relação "pertence a" como o nome já diz, é a parte do +1, tipo se for usado com hasMany seria o +1 do n+1
Technology hasMany ReviewTechnology => Technology tem muitas Reviews
E ReviewTechnology belongsTo Technology => Uma Review pertence a uma Tecnology
O belongsToManyy só serve para n+n